mentoring/public/js/mentoring_assessment_view.js

```diff
@@ -121,8 +152,17 @@ function MentoringAssessmentView(runtime, element, mentoring) {
 
         if (isDone())
             renderGrade();
+        post_display()
+    }
+
+    function post_display(results) {
         nextDOM.attr('disabled', 'disabled');
-        reviewDOM.attr('disabled', 'disabled');
+        if (! no_more_attempts()) {
```

Comment:
Personal preference (feel free to ignore): It would look more natural to me if the `if` branches were switched the other way around (`if (no_more_attempts()) {...} else {...}` rather than `if (!no_more_attempts()) ...`.